inception-project/inception
# #2295 Links to user guide do not work

**What's in the PR**

Adapt help links on pages to new url scheme

**How to test manually**
* Click on the blue-white question marks, a related section in the user guide should open.
